src/Models/Store.php

```diff
@@ -27,20 +27,17 @@ protected static function booting()
 
     public static function getCached(): array
     {
-        if (! $stores = config('cache.app.stores')) {
-            $stores = Cache::rememberForever('stores', function () {
-                return self::select([
-                    'store_id',
-                    'store.name',
-                    'store.code',
-                    'store.website_id',
-                    'store.group_id',
-                    'store_group.root_category_id',
-                    'store_website.code AS website_code',
-                ])->get()->keyBy('store_id')->toArray();
-            });
-            config(['cache.app.stores' => $stores]);
-        }
+        $stores = once(fn () => Cache::rememberForever('stores', function () {
```

Comment:
This feels a bit odd, maybe something like https://github.com/antriver/laravel-multi-cache can help out? Or maybe Laravel supports it already someway? `Cache::store(['array', 'redis'])->...`

Reply:
Since we are dependent on a PR being merged for it to work properly i've created a split PR https://github.com/rapidez/core/pull/583

Reply:
This has been resolved by maintaining our own version. The split PR has been merged into this branch.